docs/generated/manifests/menus.json

```diff
@@ -2730,6 +2730,31 @@
         "children": [],
         "disableCollapsible": false
       },
+      {
+        "name": "Database",
```

Comment:
unsure if this should be the heading or not, also should this be plural "Databases"?